addons/sale_management/models/sale_order_template.py

```diff
@@ -45,14 +45,43 @@ def _onchange_template_line_ids(self):
         if companies and self.company_id not in companies:
             self.company_id = companies[0]
 
+    @api.model
+    def create(self, vals):
+        record = super(SaleOrderTemplate, self).create(vals)
+        record._so_lines_translations()
+        return record
+
     def write(self, vals):
         if 'active' in vals and not vals.get('active'):
             template_id = self.env['ir.default'].get('sale.order', 'sale_order_template_id')
             for template in self:
                 if template_id and template_id == template.id:
                     raise UserError(_('Before archiving "%s" please select another default template in the settings.') % template.name)
-        return super(SaleOrderTemplate, self).write(vals)
-
+        result = super(SaleOrderTemplate, self).write(vals)
+        self._so_lines_translations()
+        return result
+
+    def _so_lines_translations(self):
```

Comment:
This method could have a clearer name I think, maybe something like `_update_lines_translations(self)` :thinking: